doc/benchmarking.md

```diff
@@ -0,0 +1,30 @@
+Benchmarking
+============
+
+Dash Core has an internal benchmarking framework, with benchmarks
+for cryptographic algorithms such as SHA1, SHA256, SHA512 and RIPEMD160. As well as the rolling bloom filter.
+
+After compiling dash-core, the benchmarks can be run with:
+`src/bench/bench_bitcoin`
```

Comment:
Missed one, should be `src/bench/bench_dash`

Reply:
Fixed :+1: